### ISSUE TITLE
Allow passing file-specific data

### ISSUE THREAD

**Issue by mirkolenz** [mirkolenz/makejinja#59]
Currently, the same data is used for every rendered file. It would be nice to additionally support template-specific data that is only used for a given file prefix. Could be useful for overriding values for certain files.

**Comment by 1300371** [comment]
Yes! It should be nice also to dynamically generate filenames. I mean to choose destination filenames.

For example, light-common.yaml becomes light-room.yaml, light-kitchen.yaml, ...

**Comment by mirkolenz** [comment]
That‘s a great idea, I‘ll keep it in mind when designing this function 😄

**Comment by mirkolenz** [comment]
I just added file-specific data in [v2.8](https://github.com/mirkolenz/makejinja/releases/tag/v2.8.0)! I thought about dynamic filenames, but there sre still some open questions from my side before I can give this a try. Could you open a new issue and add some additional details? For instance, I'm not sure what triggers the dynamic filename and how/where it would be specified in the first place. A small example would be greatly appreciated!
